arch/arm/core/isr_wrapper.S

```diff
@@ -128,12 +160,17 @@ _idle_state_cleared:
 	mov lr, r3
 #elif defined(CONFIG_ARMV7_M_ARMV8_M_MAINLINE)
 	pop {r0, lr}
+#elif defined(CONFIG_ARMV7_R)
```

Comment:
This #elif is empty, so maybe write something there..

Reply:
added comment